pkg/api/v1alpha1/nutanixmachineconfig.go

```diff
@@ -153,14 +153,6 @@ func validateNutanixMachineConfig(c *NutanixMachineConfig) error {
 		return fmt.Errorf("NutanixMachineConfig: %v", err)
 	}
 
-	if c.Spec.OSFamily != Ubuntu {
```

Comment:
instead of deleting this entire validation, can we do something like
`if c.Spec.OSFamily != Ubuntu && c.Spec.OSFamily != RedHat`

for ex: https://github.com/aws/eks-anywhere/blob/e53af28d9c3819dfba1fcac3b65a40333b7c55e2/pkg/api/v1alpha1/vspheremachineconfig.go#L87

Reply:
@ahreehong Fixed

Reply:
Does `RedHat` OSFamily cover both redhat8 and redhat9

Reply:
@abhinavmpandey08 Yeah. 
https://github.com/aws/eks-anywhere/pull/6822/files#diff-ed18f85e94add304fc61a5b5e588087980e478789e065ae3e8e2ea7a38735a75L27-R31